games-server/minecraft-server/minecraft-server-1.15.1-r1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+EGIT_COMMIT="4d1826eebac84847c71a77f9349cc22afd0cf0a1"
+
+inherit readme.gentoo-r1 java-pkg-2
+
+DESCRIPTION="The official server for the sandbox video game"
+HOMEPAGE="https://www.minecraft.net"
+SRC_URI="https://launcher.mojang.com/v1/objects/${EGIT_COMMIT}/server.jar -> ${P}.jar"
+
+LICENSE="Mojang"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	acct-group/minecraft
+	acct-user/minecraft
+	app-misc/dtach
+	sys-apps/openrc
```

Comment:
What's the purpose of adding ```sys-apps/openrc``` into ```RDEPEND```? Since it's Gentoos default RC system, you don't have to add it.